rm.ExtensionsTest/DateTimeExtensionTest.cs

```diff
@@ -37,5 +37,29 @@ public void AsUtcKind01()
 			Assert.AreEqual(expectedDate, date.ToUniversalTime());
 			Assert.AreEqual(expectedDate.Kind, date.ToUniversalTime().Kind);
 		}
+
+		[Test]
+		public void ToOrdinal()
```

Comment:
The existing convention is to call tests as `Method01`, `Method02`, etc. (which I understand is not descriptive) where `Method()` is the extension method. This test does not follow that convention.